Revit_Core_Engine/Query/ParameterGroupTypeId.cs

```diff
@@ -0,0 +1,95 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2025, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using Autodesk.Revit.DB;
+using System.ComponentModel;
+using BH.oM.Base.Attributes;
+#if !(REVIT2020 || REVIT2021 || REVIT2022)
```

Comment:
Why would these usings be harmful under, say, Revit 2023?

Reply:
Using one of the favourite terms of @al-fisher, I have a feeling that tail starts to wag the dog here 😉 personally I am strongly against adding a compiler directive to fix an issue with VS setup - it causes unnecessary noise ('why is it here?') plus create a really poor precedent for the future behaviours.

Is there a chance you could tweak the IDE settings to avoid the issue? Alternatively you could fully qualify the types, but that won't look great either...